### DISCUSSION TITLE
Please share your Bunches!

### DISCUSSION THREAD

**Discussion by ttscoff** [ttscoff/bunch-docs#238] [Show and tell]
I know there are a lot of Bunch users. I see the downloads, I get the feature requests, I even see pretty regular donations. What I'd LOVE to see is some of your own Bunches, even if they're simple. If you've ever solved a problem with Bunch or done anything you think others could benefit from, please share. You can post them in this discussion, create new discussions, create a Gist, whatever works. Having them in the forum will give new and old users alike some inspiration, and I'll add any standout ones to the example bunches in the documentation (with attribution, of course).

**Comment by Younghart** [comment]
I'd like to share a simple example Bunch for screen recording that relies on preinstalled apps. It asks which app should be recorded, clears the screen, then gets QuickTime Player ready to record. 
```
---
title:🖥 Screencast
---

# Clear the desktop
(hide dock)
(hide desktop)
(screen #FFFFFF)

# Hide all active apps
@@

# ask for selection of app to record (simple list, replace or extend with app names)
mainActor = ?[
	Stocks
	Chess
	Grapher
] "Which App do you want to record?"

# Open and focus selected app
${mainActor}^

# Start Quicktime Player
QuickTime Player
# close default "open" dialog, and show screen recording tools
- {cmd-. ctrl-cmd-n}
```
![App Selection Pop-up](https://user-images.githubusercontent.com/2934471/188262013-1e173523-bfcc-4f91-a4f5-de7a130ca860.png)

**Reply by ttscoff** [reply]
Excellent, thanks for sharing!

**Comment by Younghart** [comment]
I fiddled around with the Shortcuts integration, here's the result: A Bunch that asks for a project name, then initiates a shortcut to create a folder, some files, and a new Bunch to open all of those. Also logs its activity to a text file – all within the Documents folder. Customize at your heart's content!
```
---
title: New project …
shortcut: Cmd-n
menu order: 101
toggles: false
---
# Example Bunch to show Shortcut integration


# Asks for project name
project = ?"How should the new project be called?"

# Needs a shortcut called "Project" – Downloadable version at:
# https://www.icloud.com/shortcuts/d98daf615a0a467a840f16166f70b108
# The example shortcut (only useable on macs) creates a folder, a markdown and a taskpaper document, 
# and a Bunch to open all those. Also logs time and date of project creation.
(shortcut Project "${project}")
```

**Comment by Younghart** [comment]
## Find System Icons

I'm periodically looking for icons from macOS helper applications, services, mouse pointer variants etc. There are three locations in macOS I found useful. This Bunch opens them in three Finder tabs (or 3 separate Finder windows if your language selection is different from English):
```
---
title: 🔍 look for Icons
---

Finder
- /System/Library/CoreServices/
-> /System/Library/CoreServices/CoreTypes.bundle//Contents/Resources/
-> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/HiServices.framework/Versions/A/Resources/cursors

```
